doc/api/errors.md

```diff
@@ -1847,6 +1847,26 @@ Creation of a [`zlib`][] object failed due to incorrect configuration.
 A module file could not be resolved while attempting a [`require()`][] or
 `import` operation.
 
+<a id="legacy-nodejs-error-codes"></a>
+## Legacy Node.js Error Codes
+
+<a id="ERR_FS_WATCHER_ALREADY_STARTED"></a>
+### ERR_FS_WATCHER_ALREADY_STARTED
+<!-- YAML
+added: v10.0.0
```

Comment:
It would be nice if we add a "removed: v10.x.x". I guess that might not be parsed properly tough. @vsemozhetbyt do you have a suggestion for this?

Reply:
We may add the case for removing to adding and deprecating. It seems these to function need to be updated accordingly:

https://github.com/nodejs/node/blob/2bea9cefbc10ed1dd497bbae61c07d971da287dd/tools/doc/common.js#L14

https://github.com/nodejs/node/blob/2bea9cefbc10ed1dd497bbae61c07d971da287dd/tools/doc/html.js#L258

(Just grepping `added` or `deprecated` and adding a similar case may do)

Reply:
@vsemozhetbyt added these sections, now shows up as 
![screen shot 2018-08-07 at 10 34 04 am](https://user-images.githubusercontent.com/2033712/43755428-7b57abb6-9a2d-11e8-9198-acad7da81a3e.png)

Although cc: @joyeecheung , because it seems to me he has some reservations against saying "removed in v10.x.x" .

Reply:
Also, should there be a short explanation on why this error code was removed? Something on the lines of 
> Like `fs.watchFile()`, this situation is now silently ignored. 

or some such.

Reply:
@SirR4T I think it's OK to have `removed in`, but the version number should be accurate. In the case of `ERR_FS_WTCHER_ALREADY_STARTED`, it has never been released in any version, since the commit that added it (https://github.com/nodejs/node/commit/6c25f2ea49c2521dfd2423bf3a06222633ec4dc9) and the commit that removed it (https://github.com/nodejs/node/commit/301f6cc553bd73cfa345ae7de6ee81655efc57d0) are both present from v10.0.0...v10.8.0 - which means when v10.0.0 was out, the error was not even there.

Reply:
If we do add a short explanation, we can make this in `changes` blocks.

Reply:
got it. If the error codes never actually touched the releases, what should the description look like? is having both `added: v10.0.0` and `removed: v10.0.0` ok?

Reply:
@joyeecheung I have eyeballed all the versions ([using this](https://github.com/nodejs/node/issues/22061#issuecomment-409912968)) for [all the commits](https://github.com/nodejs/node/pull/22100#issuecomment-414991110) where these error codes were added or removed. I can see only `v10.9.0-v10.0.0` for all of them, which I'm guessing means none of them touched any releases.

Still not sure what the docs yaml should show, though, for these error codes.

Reply:
@joyeecheung : also, I have added versions added and versions removed, for all the commits which either added or removed the error codes. Eyeballing the versions, it seems to me that the wherever github mentions `vA.a.a .. vB.b.b`, we should be mentioning the `vB.b.b` in the docs. (For both added, as well as removed.) will that be correct?

Reply:
I think annotate them as `added: v10.0.0 removed: v10.0.0` is quite confusing. A note in the docs about that would be enough, maybe something like:

```
This error code has never been released and only existed in nightly builds.
```